R/build-news.R

```diff
@@ -147,18 +147,38 @@ data_news <- function(pkg = ".") {
   html <- markdown_body(path(pkg$src_path, "NEWS.md"), pkg = pkg)
```

Comment:
I think it's worth refactoring to pull this out in order to make testing easier — then you could inline the markdown in the rests, rather than having to create a complete package for each test.